build-concourse/README.md

```diff
@@ -128,25 +128,97 @@ Upload the following (tweak it to change which jobs are pulled) by visiting http
     headers:
       Content-Type: application/json
     body: >
-        {
-            "collection_id": "philschatz/tiny-book/book-slug1",
-            "version": null,
-            "style": "astronomy",
-            "content_server": null
-        }
+      {
+        "status_id": "1",
+        "job_type_id": "3",
+        "version": "18ca7dfc49392e2cb93352c528a140e7d02a030c",
+        "worker_version": "1",
+        "id": "3333",
+        "created_at": "2022-10-24T22:44:09.845510",
+        "updated_at": "2022-10-24T22:44:09.845515",
+        "status": {
+          "name": "assigned",
+          "id": "2"
+        },
+        "repository": {
+          "name": "osbooks-college-success",
+          "owner": "openstax"
+        },
+        "job_type": {
+          "name": "git-web-hosting-preview",
+          "display_name": "Web Preview (git)",
+          "id": "4"
+        },
+        "user": {
```

Comment:
Wow, that's a lot of extra JSON... is that all used by the resource or is this API endpoint now shared between concourse and the UI?

If it's shared, are all of these fields necessary for concourse?

Reply:
It has always been shared. In the past, the saved responses in the README omitted much of the included information. When I updated the README, I thought I wanted to use data that was true-to-life.